### ISSUE TITLE
Can't delete sequence

### ISSUE THREAD

**Issue by joeybab3** [kartaview/openstreetcam.org#349]
I can't delete a sequence, it shows this error message:
![image](https://user-images.githubusercontent.com/2829457/149264365-70299e31-89a4-4e78-9735-cec7d098c05f.png)
![image](https://user-images.githubusercontent.com/2829457/149264400-68233f34-6a54-4d20-81bb-aa1ede7bf75b.png)

**Comment by JoshuaDietz** [comment]
I have the same problem i.e. for deleting a single photo from a sequence on kartaview.org
I debugged a bit and the reason is that for these POST endpoints the access token should be provided as a normal field in the request body. Instead the application provides it as a url parameter like for GET requests, i.e. /some-endpoint/?access_token=xxxxxx so for the server the user is not authenticated.

When I sent the same request as the application sent but added the access_token in the request body the deletion worked.

**Comment by brmonaghan** [comment]
Same here. I got this message 
![still not working](https://user-images.githubusercontent.com/71337477/154077042-0e89395e-9cd8-4427-855e-10e98cae2590.png)

**Comment by joeybab3** [comment]
As a temporary workaround, you can use the app (though loading the profile sequences takes minutes or longer) to delete the sequences

**Comment by miruna-craciun** [comment]
hey, this has been fixed as of yesterday. please try to see if it works!

**Comment by brmonaghan** [comment]
![Screenshot 2022-02-17 081420](https://user-images.githubusercontent.com/71337477/154489179-a1d58033-53bd-4e48-a9d5-0eb94c7e0b35.png)
Still not working with https://kartaview.org/details/1566873
https://kartaview.org/details/1566193
https://kartaview.org/details/1456702
https://kartaview.org/details/1455626

**Comment by miruna-craciun** [comment]
@brmonaghan - you are trying to access sequences that are processing. The **delete** function can only be used if recordings are published. However, @zhaitianduo @salabogdan - the recordings mentioned above are from 2019 but can not be accessed.

**Comment by joeybab3** [comment]
I am able to delete sequences again for what it's worth. I can leave this issue open for now until we confirm that all sequences are able to be processed again.

**Comment by brmonaghan** [comment]
I tried to do that @miruna-craciun and it is still not working. This has been a problem when trying to delete sequences. But its been awhile, so let me try again.

**Comment by brmonaghan** [comment]
Aaand its now working! @joeybab3 you can now close this.
THANK YOU VERY MUCH @zhaitianduo AND @miruna-craciun FOR FIXING THIS!

**Comment by nakic** [comment]
> @brmonaghan - you are trying to access sequences that are processing. The **delete** function can only be used if recordings are published. However, @zhaitianduo @salabogdan - the recordings mentioned above are from 2019 but can not be accessed. Can you take a look?

I have tracks which show up "processing" but will never process and cannot be deleted, e.g. this one: https://kartaview.org/details/4537689. I don't see a way to either change its status or delete it. Am I missing something?

**Comment by brmonaghan** [comment]
agree.
